SharedPackages/BrowserServicesKit/Sources/Subscription/Managers/V1toV2Bridging/SubscriptionAuthV1toV2Bridge.swift

```diff
@@ -54,15 +61,19 @@ public protocol SubscriptionAuthV1toV2Bridge: SubscriptionTokenProvider, Subscri
 }
 
 extension SubscriptionAuthV1toV2Bridge {
-
-    public func isEnabled(feature: Entitlement.ProductName) async throws -> Bool {
-        try await isFeatureAvailableAndEnabled(feature: feature, cachePolicy: .returnCacheDataElseLoad)
+    public func isFeatureIncludedInSubscription(_ feature: Entitlement.ProductName) async throws -> Bool {
+        try await currentSubscriptionFeatures().contains(feature)
```

Comment:
I think this should be the correct way to to check a feature is available within the subscription.  Can you verify it checks out?